code/src/java/plugin/modifier/set/AddModifierFactory.java

```diff
@@ -121,16 +122,16 @@ public String getInstructions()
 
 	}
 
-	public class AddIndirectArrayModifier extends AddArrayModifier
+	private final class AddIndirectArrayModifier extends AddArrayModifier
 	{
 		/**
 		 * The objects to be added to the active set when this AddModifier is
 		 * processed
 		 */
 		private Indirect<T[]> toAdd;
 
-		public AddIndirectArrayModifier(FormatManager<T[]> formatManager,
-			int userPriority, Indirect<T[]> toAdd)
+		private AddIndirectArrayModifier(FormatManager<T[]> formatManager,
+		                                 int userPriority, Indirect<T[]> toAdd)
```

Comment:
I have to admit I don't know the exact knob, which is why we have the standards file for formatting.